### PR TITLE
[SRM-982] Updated profile to be compatible with drupal 10.

### PR DESCRIPTION
**Pull request by MdNadimHossain**
### JIRA 
https://digital-vic.atlassian.net/browse/SRM-982

### Changes
Removed deprecated modules and unused modules from tide profile. Updated modules to be D10 compatible version.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,33 +1,29 @@
...
     "type": "drupal-profile",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_alert": "3.0.2",
-        "dpc-sdp/tide_api": "3.0.14",
-        "dpc-sdp/tide_authenticated_content": "3.0.8",
-        "dpc-sdp/tide_core": "3.2.14",
-        "dpc-sdp/tide_demo_content": "3.0.13",
-        "dpc-sdp/tide_event": "3.0.4",
-        "dpc-sdp/tide_event_atdw": "3.0.1",
```
Comment (MdNadimHossain) [inline composer.json:13]: Not in use anymore.

File: composer.json [hunk 2]
```diff
@@ -1,33 +1,29 @@
 {
     "name": "dpc-sdp/tide",
-    "description": "Tide Drupal 9 profile",
+    "description": "Tide Drupal 10 profile",
     "type": "drupal-profile",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_alert": "3.0.2",
-        "dpc-sdp/tide_api": "3.0.14",
-        "dpc-sdp/tide_authenticated_content": "3.0.8",
```
Comment (MdNadimHossain) [inline composer.json:9]: Only SSP using it. Recently got removed from vicpol. So taking this out of tide profile.

File: composer.json [hunk 3]
```diff
@@ -1,33 +1,29 @@
...
-        "dpc-sdp/tide_core": "3.2.14",
-        "dpc-sdp/tide_demo_content": "3.0.13",
-        "dpc-sdp/tide_event": "3.0.4",
-        "dpc-sdp/tide_event_atdw": "3.0.1",
-        "dpc-sdp/tide_grant": "3.0.5",
-        "dpc-sdp/tide_landing_page": "3.0.16",
-        "dpc-sdp/tide_media": "3.0.13",
-        "dpc-sdp/tide_monsido": "3.0.1",
```
Comment (MdNadimHossain) [inline composer.json:17]: Deprecated and now monsido controlled by GTM.

**Review by vincent-gao — APPROVED** [review]

**Comment by vincent-gao** [comment]
🚀

**Pull request merged**